assets/src/dashboard/app/views/previewStory/index.js

```diff
@@ -62,11 +62,6 @@ const IframeContainer = styled.div`
   width: ${({ dimensions }) => `${dimensions.width}px`};
   min-height: 90vh;
   height: calc(100% - ${THEME_CONSTANTS.WP_ADMIN.TOOLBAR_HEIGHT}px);
```

Comment:
This focus border wouldn't show in any browser, so I just removed the style.

Reply:
previewStory is getting axed this sprint, just heads up - so no need to have QA test that aspect for regression (it's also hidden behind a feature flag that should be off 😅 ).